client/src/details/contactdetails.cpp

```diff
@@ -116,6 +116,12 @@ void ContactDetails::initialize()
 
     connect(mUi->primaryAddressCountry, SIGNAL(editingFinished()), SLOT(slotPrimaryAddressCountryEditingFinished()));
     connect(mUi->altAddressCountry, SIGNAL(editingFinished()), SLOT(slotOtherAddressCountryEditingFinished()));
+    connect(mUi->copyPrimaryAddressButton, &QPushButton::clicked, [this]() {
```

Comment:
It's good practice to always have a receiver as 3rd argument before the lambda, so that the connection gets disconnected when the receiver goes away. I agree that here it can't happen, but still, I suggest to take the habit of always specifying a receiver, it's just too easy to fall into that trap otherwise.  `, this` will do.

Reply:
Wow, this is nice to know. Thanks for the tip. 
Fixed